src/applicants.js

```diff
@@ -0,0 +1,45 @@
+import applicantApi from './applicant-api.js';
+
+const tbody = document.getElementById('applicants');
+
+const applicants = applicantApi.getAll();
+
+for(let i = 0; i < applicants.length; i++) {
+    const applicant = applicants[i];
+    console.log(tbody);
+    const tr = document.createElement('tr');
+
+    const nameCell = document.createElement('td');
+    nameCell.textContent = applicant.name;
+    tr.appendChild(nameCell);
+
+    const phoneCell = document.createElement('td');
+    phoneCell.textContent = applicant.phone;
+    tr.appendChild(phoneCell);
+
+    const brandCell = document.createElement('td');
+    brandCell.textContent = applicant.brand;
+    tr.appendChild(brandCell);
+
+    const descriptionCell = document.createElement('td');
+    descriptionCell.textContent = applicant.description;
+    tr.appendChild(descriptionCell);
+
+    const sizeCell = document.createElement('td');
+    sizeCell.textContent = applicant.size;
+    tr.appendChild(sizeCell);
+
+    /*const futuresCell = document.createElement('td');
```

Comment:
Try not leave commented out code in your PR when you submit. If you aren't using it you can just delete it.